tests/integrations/test_db_integrations_sql.py

```diff
@@ -417,11 +496,42 @@ def test_obfuscate_query(self):
             elif db_operation == "procedure":
                 # Insert and procedure:These operations also receive two parameters, but are obfuscated as only one.
                 # Nodejs: The proccedure has a input parameter, but we are calling through method execute and we can't see the parameters in the traces
-                expected_obfuscation_count = 0 if context.library.library == "nodejs" else 1
+                expected_obfuscation_count = 0 if context.library.library == "nodejs" else 2
             else:
                 expected_obfuscation_count = 2
 
             observed_obfuscation_count = span["meta"]["sql.query"].count("?")
             assert (
                 observed_obfuscation_count == expected_obfuscation_count
             ), f"The mssql query is not properly obfuscated for operation {db_operation}, expecting {expected_obfuscation_count} obfuscation(s), found {observed_obfuscation_count}:\n {span['meta']['sql.query']}"
+
+
+@scenarios.otel_integrations
+class Test_Agent_Mssql_db_otel_integration(_BaseOtelAgentIntegrationsSqlTestClass, _Base_Mssql_db_integration):
+    """ Overwrite or add specific validation methods for mssql on agent interface (app instrumented by open telemetry) """
+
+    def test_db_operation(self):
+        """ The name of the operation being executed. Mssql and Open Telemetry doesn't report this span when we call to procedure """
+        for db_operation, request in self.requests[self.db_service].items():
+            span = self._get_sql_span_for_request(request)
```

Comment:
Your request the span even for `select_error`, `procedure`, even if you check nothing, is it on purpose ? If yes, maybe a small comment can help futur readers.